staging/src/k8s.io/sample-controller/README.md

```diff
@@ -92,8 +92,8 @@ kubectl create -f artifacts/examples/crd.yaml
 # create a custom resource of type Foo
 kubectl create -f artifacts/examples/example-foo.yaml
 
-# check deployments created through the custom resource
-kubectl get deployments
```

Comment:
https://github.com/kubernetes/kubernetes/blob/master/staging/src/k8s.io/sample-controller/controller.go#L243-L319， deployment created by cr Foo, so change in PR seems not correct